@vates/parse-duration/index.js

```diff
@@ -6,6 +6,9 @@ exports.parseDuration = value => {
   if (typeof value === 'number') {
     return value
   }
+  if (!(typeof value === 'string' && value !== '')) {
```

Comment:
```suggestion
  if (typeof value !== 'string' || value === '') {
```
I feel it's more clear to avoid double negation

Reply:
Done.